fantiadl/fantiadl.py

```diff
@@ -60,6 +60,10 @@ def main():
 
     if not (cmdl_opts.download_fanclubs or cmdl_opts.download_paid_fanclubs or cmdl_opts.download_new_posts) and not cmdl_opts.url:
         sys.exit("Error: No valid input provided")
+    if cmdl_opts.download_new_posts and cmdl_opts.download_fanclubs:
+        sys.exit("Error: -n/--download-new-posts and -f/--download-fanclubs are mutually exclusive")
```

Comment:
`Error: -n/--download-new-posts and -f/--download-fanclubs are mutually exclusive. Please specify only one of these flags.`